Gemfile

```diff
@@ -2,6 +2,10 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 gemspec
 
+gem "byebug"
+gem "ruby-debug-ide"
+gem "debase"
+
```

Comment:
Best to put these in `Gemfile.local` and this `Gemfile` will load it automatically